README.md

```diff
@@ -1,5 +1,6 @@
 
 [![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg?logo=zulip)](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system)
+[![project chat](https://img.shields.io/badge/Docusauras%20-2.0.0--alpha.72-brightgreen)](https://anitab-org.github.io/mentorship-android/)
```

Comment:
```suggestion
[![Documentation](https://img.shields.io/badge/documentation-mentorship--android-green.svg)](https://anitab-org.github.io/mentorship-android/)
```

Docusaurus was had a typo, but I thought this could be similar to what we have in mentorship-backend repository :) Could you please update this?